pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/OffloadPolicies.java

```diff
@@ -51,8 +51,8 @@
     private String managedLedgerOffloadDriver = null;
     private int managedLedgerOffloadMaxThreads = DEFAULT_OFFLOAD_MAX_THREADS;
     private int managedLedgerOffloadPrefetchRounds = DEFAULT_OFFLOAD_MAX_PREFETCH_ROUNDS;
-    private long managedLedgerOffloadThresholdInBytes = DEFAULT_OFFLOAD_THRESHOLD_IN_BYTES;
-    private Long managedLedgerOffloadDeletionLagInMillis = DEFAULT_OFFLOAD_DELETION_LAG_IN_MILLIS;
+    private long managedLedgerOffloadAutoTriggerSizeThresholdBytes = DEFAULT_OFFLOAD_THRESHOLD_IN_BYTES;
```

Comment:
What is the backward compatibility story here because we are changing the field name for `OffloadPolicies`? If there is already a namespace policy written using previous field names, how do we guarantee the new code is able to read the offload policies written using the old code?

Reply:
Yes, this change will cause the compatible problem, I resolved this problem.